### ISSUE TITLE
StringToString usage print is unstable

### ISSUE THREAD

**Issue by SOF3** [spf13/pflag#362]
Since Go maps have randomized iteration order, `FlagUsages()` returns inconsistent output:

https://go.dev/play/p/NYW3Gr3c6ic

In projects with a workflow that writes the command usage to a file (e.g. README), this causes unnecessary diff all the time.

**Comment by twz123** [comment]
It's a general issue concerning reproducibility. If you want to create an OS package including docs, say, man pages, it won't be reproducible since the man pages will differ on each build.

**Comment by cornfeedhobo** [comment]
lol. I flagged this issue over 3 years ago. Fixed it on my fork and submitted a PR. Straight up - this repo is not maintained and the owners do not seem capable of owning up to this.

**Comment by tomasaschan** [comment]
This issue has not been updated in a long time. Is it still relevant?

(If I receive no answer before the end of the summer, I'll close it. Feel free to re-open if you come here after that happens and it is relevant to you - but if you do, please also comment and specify what outcome you're interested in!)

**Comment by SOF3** [comment]
Yes, you can verify again by running the linked example above on playground.

**Comment by twz123** [comment]
> specify what outcome you're interested in

Reproducible output, i.e. so that the help text is always the same for each invocation of a binary. An obvious fix could e.g. just sort the map keys when creating the usage string.
